### ISSUE TITLE
NTR: Succinylacetone high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8577]
New term request Succinylacetone high in urine (segal_030509153957):   UMLS: 1848700

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000598
- label: Elevated urinary succinylacetone level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: PMID:23311542
- def: Presence in the urine of succinylacetone, which is one of the toic block metabolites formed in tyrosinaemia type 1 because of a defect in the final enzyme of the pathway of the degradation of tyrosine, namely fumarylacetoacetase (FAH, EC 3.7.1.2).
- comment: As a result of the metabolic block in tyrosinaemia type 1, toxic metabolites are formed including succinylacetone, maleylacetoacetate and fumarylacetoacetate. These are responsible for severe disruption of intracellular metabolism of the liver and kidney.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8577
- synonyms:
